test/setup.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const chai = require('chai');
+
+global.sinon = require('sinon');
+global.assert = chai.assert;
```

Comment:
Why on earth, you decided to make them global? Why not move them to `test/configurable.js`?